src/main/scala/org/broadinstitute/dsde/agora/server/ServerInitializer.scala

```diff
@@ -32,8 +36,18 @@ class ServerInitializer(val config: Config) extends LazyLogging {
   }
 
   private def startWebServiceActors() {
+    implicit val timeout = Timeout(5.seconds)
+    import scala.concurrent.ExecutionContext.Implicits.global
     val service = actorSystem.actorOf(ApiServiceActor.props, "agora-actor")
-    IO(Http) ! Http.Bind(service, interface = webserviceInterface, port = webservicePort)
+    (IO(Http) ? Http.Bind(service, interface = webserviceInterface, port = webservicePort)).onComplete {
+      case Success(Http.CommandFailed(failure)) =>
```

Comment:
My only comment is about the Success(failure) type of pattern here. Until we start getting used to that as a group I'd either comment that this is intentional and why it happens or set up some structure to split the two apart